doc/sphinx/source/buildlb.rst

```diff
@@ -62,7 +66,7 @@ as needed:
    -- NGRAPH_CODE_COVERAGE_ENABLE:     OFF
    -- NGRAPH_LIB_VERSIONING_ENABLE:    OFF
    -- NGRAPH_PYTHON_BUILD_ENABLE:      OFF
-   -- NGRAPH_USE_PREBUILT_LLVM:        OFF
+   -- NGRAPH_USE_PREBUILT_LLVM:        FALSE
```

Comment:
I don't understand this change. OFF was good.

Reply:
I have no idea how this happened.  Will revert.